reek.yml

```diff
@@ -9,4 +9,8 @@ InstanceVariableAssumption:
 UncommunicativeMethodName:
   enabled: false
```

Comment:
@marderer did you try to enable `UncommunicativeMethodName`?

Reply:
@AleksSenkou yes, I try, but I don't know what do with `google_oauth2`

Reply:
@marderer try `config.omniauth :google_oauth2, "GOOGLE_CLIENT_ID", "GOOGLE_CLIENT_SECRET", { name: "google" }`

Reply:
`UncommunicativeMethodName` is super important metric